week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,31 +1,40 @@
-// Add comments to explain what this function does. You're meant to use Google!
+//This fuction returns a floating-point (decimal) random number, from 0 to 1 (inclusive of 0, but not 1) and mutiply by 10.
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
 }
 
-// Add comments to explain what this function does. You're meant to use Google!
+// This function joins (concatenates) w1 and w2 strings and returns a new string containing the text of the joined strings.
 function s(w1, w2) {
-    return w1.concat(w2);
+  return w1.concat(w2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
-    // Write the body of this function to concatenate three words together
-    // Look at the test case below to understand what to expect in return 
+  return `${firstWord} ${secondWord} ${thirdWord}`;
+  // I had to use template string method here
```

Comment:
First, I tried concat method here, `return firstWord.concat(secondWord, thirdWord);`
but I could not pass the tests. I thought maybe because we can't concatenate strings and integers with concat method. Then I used string template method. If I can't figure it out myself I can ask about this when we have a call on Thursday.